frontend/src/styles/components/EmailFooter.scss

```diff
@@ -0,0 +1,104 @@
+$email-icon--width: 146px;
+$email-icon--height: 127px;
+
+$email-icon--responsive-unit: .66;
+
+.email-footer {
+  background: $transparent-white-1;
```

Comment:
OK, I was just copy-pasta-ing that from Circle-CI, so we should see if we can blow away that cache.